lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -0,0 +1,104 @@
+require 'ipaddr'
+require 'pathname'
+
+module LinuxAdmin
+  class NetworkInterfaceRH < NetworkInterface
```

Comment:
My intention was "Red Hat" as the class is able to provide network configuration for CentOS, Fedora, and RHEL.